pennylane_qrack/QrackDeviceConfig.toml

```diff
@@ -0,0 +1,154 @@
+# Which version of the specification format is being used.
+schema = 2
+
+# The union of all gate types listed in this section must match what
+# the device considers "supported" through PennyLane's device API.
+# The gate definition has the following format:
+#
+#   GATE = { properties = [ PROPS ], condition = [ COND ] }
+#
+# Where:
+#
+#   PROPS: zero or more comma-separated quoted strings:
+#          "controllable", "invertible", "differentiable"
+#   COND: quoted string, on of:
+#         "analytic", "finiteshots"
+#
+[operators.gates.native]
+
+PauliX = { properties = [ "controllable", "invertible"] }
+PauliY = { properties = [ "controllable", "invertible"] }
+PauliZ = { properties = [ "controllable", "invertible"] }
+SX = { properties = [ "controllable", "invertible"] }
+MultiRZ = { properties = [ "controllable", "invertible" ] }
+Hadamard = { properties = [ "controllable", "invertible"] }
+S = { properties = [ "controllable", "invertible" ] }
+T = { properties = [ "controllable", "invertible" ] }
+CNOT = { properties = [ "controllable", "invertible" ] }
+SWAP = { properties = [ "controllable", "invertible" ] }
+CSWAP = { properties = [ "controllable", "invertible" ] }
+ISWAP = { properties = [ "controllable", "invertible"] }
+PSWAP = { properties = [ "controllable", "invertible"] }
+Toffoli = { properties = [ "controllable", "invertible" ] }
+CY = { properties = [ "controllable", "invertible" ] }
+CZ = { properties = [ "controllable", "invertible" ] }
+PhaseShift = { properties = [ "controllable", "invertible" ] }
+ControlledPhaseShift = { properties = [ "controllable", "invertible" ] }
+CPhase = { properties = [ "controllable", "invertible" ] }
+RX = { properties = [ "controllable", "invertible" ] }
+RY = { properties = [ "controllable", "invertible" ] }
+RZ = { properties = [ "controllable", "invertible" ] }
+Rot = { properties = [ "controllable", "invertible" ] }
+CRX = { properties = [ "controllable", "invertible" ] }
+CRY = { properties = [ "controllable", "invertible" ] }
+CRZ = { properties = [ "controllable", "invertible" ] }
+CRot = { properties = [ "controllable", "invertible" ] }
+U3 = { properties = [ "controllable", "invertible" ] }
+U2 = { properties = [ "controllable", "invertible" ] }
+U1 = { properties = [ "controllable", "invertible" ] }
+MultiControlledX = { properties = [ "controllable", "invertible" ] }
+Identity = { properties = [ "controllable", "invertible" ] }
+QFT = { properties = [ "invertible" ] }
+
+# Operators that should be decomposed according to the algorithm used
+# by PennyLane's device API.
+# Optional, since gates not listed in this list will typically be decomposed by
+# default, but can be useful to express a deviation from this device's regular
+# strategy in PennyLane.
+# [operators.gates.decomp]
+
+# QubitUnitary = {}
```

Comment:
I suppose we want to keep these here for now, as the next step is to add operator support. Perhaps we can add a comment with a referenced task about that.

Reply:
Yes, let's keep this section (commented), since it contains basically all remaining standard gates and operations. This way, we just shuffle around the list with new implementation, and we're not wondering what the basic operations are.